scripts/src/blockchain/txSender.ts

```diff
@@ -92,13 +90,9 @@ export class TxSender {
 						// so there is no need to build the transaction again
 						return this.submitTransaction(builder);
 					}
-					throw new TransactionNotFoundError(this._keypair.publicAddress);
-				} else {
-					throw new ServerError(e.response.status, e.response);
 				}
-			} else {
-				throw new NetworkError(e.message);
 			}
+			throw new TranslateError(e);
```

Comment:
the above check `checkForInsufficientChannelFeeBalance` should be refactored now to use `TranslateError`, i.e. first translate and then check against the translated error if it's the error we looking for. (as now we "translating" the error twice)